### ISSUE TITLE
As a council employee, I want to be able to add a resource with a link to some information to be considered for display to FOI requesters so that no one else will make a request for that information

### ISSUE THREAD

**Issue by garethrees** [mysociety/foi-for-councils#26]
**Essential - dev**
- [x] Flash messages for success & failure
- [x] Review where we redirect after each action
- [x] Way to get back to the index page
- [x] ~~Implement Postgres search matching `CuratedLink#keywords` to `FoiRequest#body`~~ #105 

**Essential - design**
- [x] Flash message styles (success, error, see 80c7fb3a513bd219956e2c1e3a6fa65bcc99310c) https://trello.com/c/IdXKgqNL/127-foifc-flash-message-styles
- [x] Style user-displayed suggestions as per prototype
- [x] Front end library for converting keywords field to CSV

**Nice to have**

Extracted to https://github.com/mysociety/foi-for-councils/issues/123

**Comment by garethrees** [comment]
Waiting on feedback

**Comment by garethrees** [comment]
This is "done", pending any major blockers from their end.

I've extracted the group by category nice-to-haves in to https://github.com/mysociety/foi-for-councils/issues/123.

**Comment by garethrees** [comment]
They've confirmed that they're happy with the direction. The main thing they commented on was lack of form labels/examples, which is in progress in https://github.com/mysociety/foi-for-councils/pull/114.

Closing this in favour of any specific tickets that come up.
